fwk.c

```diff
@@ -1170,23 +1170,41 @@ float audio_volume_master(float gain) {
     return sqrt( volume_master );
 }
 
+int audio_play_gain_pitch_pan( audio_t a, int flags, float gain, float pitch, float pan ) {
+    // gain: 0..+1
+    // pitch: 0..N
+    // pan: -1..+1
+
+    if (!(flags & AUDIO_OVERRIDE_GAIN)) {
+        gain = a->is_clip ? volume_clip : volume_stream + gain;
```

Comment:
yes the idea is that you could specify custom gain for specific sounds if override is enabled, otherwise it performs addition so that you can fine-tune individual sounds on top of global volume setting.

Reply:
oh yes sorry, ye this is definitely a typo.